Lib/typing.py

```diff
@@ -30,6 +30,7 @@
 import sys
 import types
 from types import WrapperDescriptorType, MethodWrapperType, MethodDescriptorType, GenericAlias
+import inspect
```

Comment:
that's going to do very bad things to the import time of `typing.py`

Reply:
I've moved it to just use `types.FunctionType` instead.